src/app/frontend/deploy/deploy.scss

```diff
@@ -12,19 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.kd-upload-button {
-  padding-left: 0;
-  padding-right: 0;
-}
-
-.kd-upload-label {
-  cursor: inherit;
-  display: inline-block;
-  width: 100%;
-}
+@import '../variables';
 
-.kd-upload-file-picker {
-  height: 0;
-  visibility: hidden;
-  width: 0;
+.kd-deploy-tabs {
+  md-tab-content {
+    padding: 2.5 * $baseline-grid;
+  }
```

Comment:
I'd remove transition effect when switching tabs. It's not that fluent. It can be removed by using:
```css
[role="tabpanel"] {
  transition: none;
}
```

Reply:
Sure, it looks even better.